code/modules/research/nodes/combat.dm

```diff
@@ -239,7 +239,8 @@
 							/datum/design/research/item/ammo/heavy_rifle_408,
 							/datum/design/research/item/ammo/laser_mag,
 							/datum/design/research/item/weapon/laser_ak,
-							/datum/design/research/item/weapon/centauri)
+							/datum/design/research/item/weapon/centauri,
+							/datum/design/research/item/weapon/heavy_plasma)
```

Comment:
Leave this out untill you get the sprite

Reply:
It currently got the same sprite as the Purger.

Reply:
untill you get a new sprite for it then add it into rnd

Reply:
Hound is currently working on the sprite.

Reply:
And untill they are done do not have it in rnd ndes

Reply:
I understand why you don't want it, lore and everything, but I am not going to make a commit to remove it from RnD, only to revert it **again** once the sprite is done.
Beside, I doubt @Kazkin will merge this PR before the sprite is done. if he does, you can yell at him.

Reply:
So I asked Kaz, and it is commented out until the sprite is added....